src/NzbDrone.Common/Http/HttpClient.cs

```diff
@@ -28,11 +28,14 @@ public class HttpClient : IHttpClient
         private readonly ICached<CookieContainer> _cookieContainerCache;
         private readonly ICached<bool> _curlTLSFallbackCache;
 
+        public bool forceCurl { get; set; }
```

Comment:
Yes, it's just for testing.  It seemed like the easiest way to be able to re-use all the existing tests without having to modify them.  I guess I could make forcing curl a property of the request rather than the client?  I'm no expert so definitely open to suggestions!